### PR TITLE
[Content Manager] Add the new JSONInput from the DS to the CM

### PR DESCRIPTION
**Pull request by markkaylor**
### What does it do?

- Replaces the current json input component with the one migrated to the design system
- Removes the old json input from the monorepo

### Why is it needed?

- To completely replace the old input

### How to test it?

- Open a content-type with a JSON field
- You should see the new JSONInput
- It should behave the same as the old input

Screenshot 2023-01-27 at 11 48 59

### CODE REVIEW

File: packages/core/helper-plugin/lib/src/components/GenericInput/index.js [hunk 1]
```diff
@@ -134,6 +136,23 @@ const GenericInput = ({
...
+          labelAction={labelAction}
+          value={value}
+          error={errorMessage}
+          hint={hint}
+          required={required}
+          onChange={(json) => {
+            onChange({ target: { name, value: json } });
+          }}
+          minHeight={pxToRem(253)}
+          maxHeight={pxToRem(506)}
```
Comment (gu-stav) [inline packages/core/helper-plugin/lib/src/components/GenericInput/index.js:152]: @maevalienard Should we use that moment to align the JSON Input to our 4point grid? :) Would you have input on how we could improve these values?

**Review by gu-stav — COMMENTED** [review]
I can't tell you how happy this makes me :D Would you mind adding a test to the `GenericInput` to prove it renders the JSONInput for json fields?

**Review by joshuaellis — APPROVED** [review]
Thanks Mark!! 🤩

Same as Gustav, but have a tick.

File: packages/core/helper-plugin/lib/src/components/GenericInput/index.js (hunk 1 above)
Comment (maevalienard) [reply packages/core/helper-plugin/lib/src/components/GenericInput/index.js:152]: We can make it:

minHeight={pxToRem(252)}
maxHeight={pxToRem(504)}

**Comment by maevalienard** [comment]
Looks great to me! I feel like we just have a small border-radius issue here: 
Capture d’écran 2023-01-27 à 14 32 08
The background doesn't seem to fit the border properly in the corners :)

**Comment by markkaylor** [comment]
> Looks great to me! I feel like we just have a small border-radius issue here:

That's a bummer, I had noticed that a while back and thought I had fixed it. Guess not. I'll open a PR in the DS. Thanks!

**Comment by markkaylor** [comment]
Hey @gu-stav I don't think there are any dependencies I can remove since [Editor](https://github.com/strapi/strapi/blob/83a980fe937ed15e96cd98ce9154be4c56ec4c11/packages/core/admin/admin/src/content-manager/components/Wysiwyg/Editor.js#L3) still uses codemirror, I checked for anything else but let me know if you spotted something I missed

**Review by gu-stav — APPROVED** [review]
You are right - sorry. I didn't know we were using codemirror there too. Looks good to me then! 👍🏼

**Pull request merged**